src/libraries/System.Security.Cryptography/tests/OpenSslNamedKeysTests.manual.cs

```diff
@@ -76,14 +72,14 @@ public static void NullArguments()
             Assert.Throws<ArgumentNullException>("keyId", () => SafeEvpPKeyHandle.OpenPublicKeyFromEngine(TestEngineName, null));
         }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsOpenSslSupported))]
```

Comment:
If I run these tests on macOS without OpenSSL, they abort the whole test run.

> [xUnit.net 00:00:00.88]
> System.Security.Cryptography.Tests.OpenSslNamedKeysTests.Tpm_SanityTest [SKIP]
The active test run was aborted. Reason: Test host process crashed : No usable version of libssl was found

These tests need to be skipped on macOS that does not have OpenSSL installed since it is an optional component. I think you are looking for `PlatformDetection.OpenSslPresentOnSystem` instead of `IsOpenSslSupported` (and undo the change to `IsOpenSslSupported`)

Reply:
Seems OpenSslPresentOnSystem throws on Windows - will send a patch for that shortly...